client/components/GameDescription.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const GameDescription = (props) => (
+  <div>
```

Comment:
this seems like an excessive amount of 'divs'. Any way to refactor using less?

Reply:
Code is still in construction but I think I need all of them as class will be implemented for each div